config/locales/dynamic_form.en.yml

```diff
@@ -1,9 +1,10 @@
+---
 en:
   dynamic_form:
-    errors:
-      dynamic_format: "%{message}"
     error_messages:
+      body: 'There were problems with the following fields:'
       header:
-        one: "1 error prohibited this %{record_name} from being saved"
+        one: 1 error prohibited this %{record_name} from being saved
```

Comment:
So, what is the norm for no-quotes, single quotes, and double quotes?

All the other lines shown here have either single quotes or double quotes.  Since this has a variable in it, I would guess it needs double quotes like it originally had, but I don't know for sure.  I just know it seems inconsistent with the other message values shown.

Reply:
@MyNameIsMissing This was auto-updated by our internationalization gem `i18n-tasks` and is alright as is.  In these .yml files quotes (single or double) are used for escaping purposes only.  So, the single quotes on line 5, for example, are being used to escape the `:`. The message on ln10 is double quoted because that interpolated string *could* contain special characters (including `'` which can't be escaped inside of a single quote).